spec/views/recipes/index.html.erb_spec.rb

```diff
@@ -0,0 +1,26 @@
+require 'rails_helper'
+
+RSpec.describe 'recipes/index', type: :feature do
+  include Devise::Test::IntegrationHelpers
+  before(:example) do
+    @user = User.create(name: 'Kolly', email: 'asdasasd@gmail.com', password: 'password', confirmed_at: Time.now)
+    @recipe = Recipe.create(name: 'Recipe', description: 'Description', cooking_time: '1 hour',
+                            preparation_time: '1 hour', user: @user)
+    sign_in @user
+    visit recipes_path
+  end
+  it 'renders a list of recipes' do
+    expect(page).to have_content 'Recipe'
+  end
+
+  it 'click on new recipe' do
+    click_on 'Add new recipe'
+    expect(current_path).to eql recipes_path
+  end
```

Comment:
- [ ] This test is currently failing because the expected path does not match the actual path. Please try to make it green ✔️ 
![image](https://user-images.githubusercontent.com/46839548/234915239-f38b2666-73a0-414d-9f52-fe505756704b.png)